packages/flutter/lib/src/material/raised_button.dart

```diff
@@ -98,7 +100,7 @@ class RaisedButton extends StatelessWidget {
     this.highlightElevation = 8.0,
     this.disabledElevation = 0.0,
     this.shape,
-    this.clipBehavior = Clip.none,
+    this.clipBehavior = ui.defaultClipBehavior, // ignore: deprecated_member_use
```

Comment:
You provide the same default value for clipBehavior in RawMaterialButton, so there's no need to specify it here.

Reply:
same as above